torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10569,9 +10569,7 @@ def error_inputs_mean(op_info, device, **kwargs):
                    ref=np.arccosh,
                    domain=(1, None),
                    dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16),
-                   dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
-                   # "rsqrt_cuda" not implemented for 'BFloat16'
```

Comment:
How does this PR affect backward of `acosh` to support `BFloat16`?

Reply:
Hi, @kshitij12345! Thanks for point that out. I removed that because I got the following error:
```python
_______________________________ TestCommonCUDA.test_dtypes_acosh_cuda _______________________________
Traceback (most recent call last):
  File "/home/khushi/Documents/pytorch/test/test_ops.py", line 313, in test_dtypes
    self.fail(msg)
  File "/home/khushi/anaconda3/envs/pytorch-dev/lib/python3.10/unittest/case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: The supported dtypes for acosh on device type cuda are incorrect!
The following dtypes worked in backward but are not listed by the OpInfo: {torch.complex32}.
```
Thanks!

Reply:
Ah, I see. The comment is incorrect! `BFloat16` was present in `backward_dtypesIfCUDA`. Thanks!